src/app/shared/card/card-group-link/card-group-link.component.ts

```diff
@@ -0,0 +1,8 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-card-group-link',
+  templateUrl: './card-group-link.component.html',
+  styleUrls: ['./card-group-link.component.scss'],
+})
+export class CardGroupLinkComponent {}
```

Comment:
i don't understand the purpose of this component;
it seems to be an empty wrapper, no functionality or style applied to content

Reply:
I thought it will have some settings, but you're right. It wasn't necessary. Removed.